ext/search_kit/Civi/Api4/Action/SearchDisplay/GetSearchTasks.php

```diff
@@ -71,6 +71,13 @@ public function _run(\Civi\Api4\Generic\Result $result) {
       'number' => '>= 0',
     ];
 
+    $tasks[$entity['name']]['process'] = [
+      'module' => 'crmSearchTasks',
+      'title' => E::ts('Process Submissions'),
+      'icon' => 'fa-check-square-o',
+      'uiDialog' => ['templateUrl' => '~/crmSearchTasks/crmSearchTaskProcess.html'],
+    ];
```

Comment:
This whole thing could be moved into the afform extension. See `hook_civicrm_searchKitTasks`

Reply:
done